examples/rbd/storageclass.yaml

```diff
@@ -25,12 +34,18 @@ parameters:
    imageFeatures: layering
 
    # The secrets have to contain Ceph admin credentials.
+   # NOTE: If using "clusterID" instead of "monitors" above, the following
+   # secrets MAY be added to the ceph-cluster-<cluster-id> secret and skipped
+   # here
    csi.storage.k8s.io/provisioner-secret-name: csi-rbd-secret
    csi.storage.k8s.io/provisioner-secret-namespace: default
    csi.storage.k8s.io/node-publish-secret-name: csi-rbd-secret
    csi.storage.k8s.io/node-publish-secret-namespace: default
 
    # Ceph users for operating RBD
+   # NOTE: If using "clusterID" instead of "monitors" above, the following
+   # IDs MAY be added to the ceph-cluster-<cluster-id> secret and skipped
+   # here
    adminid: admin
```

Comment:
are we going to deprecate `admin` `monitors` `userid` from storage class in the future?